classes/class-log.php

```diff
@@ -107,6 +107,10 @@ function( $var ) {
 			$role = '';
 		}
 
+		// Support proxy mode by checking the `X-Forwarded-For` header first
+		$ip_address = wp_stream_filter_input( INPUT_SERVER, 'HTTP_X_FORWARDED_FOR', FILTER_VALIDATE_IP );
+		$ip_address = $ip_address ? $ip_address : wp_stream_filter_input( INPUT_SERVER, 'REMOTE_ADDR', FILTER_VALIDATE_IP );
```

Comment:
@fjarrett This is cool, thanks. The `REMOTE_ADDR` is also requested on line 154/158 So will need to be applied there. 
Perhaps it might be better to have method to fetch the IP, or even a simpleIP param set in the constructor to prevent code duplication.